docs/adr/006_cups.md

```diff
@@ -221,12 +221,14 @@ else *CredentialUrl is plain Url
     Function->>HTTPS Endpoint: Get Blob with HTTP GET Request
 end
 Function->>LoRaWanNetworkSrvModule: Return credential blob
-Note right of LoRaWanNetworkSrvModule: Response should be a json including base64 encoded credential
+Note right of LoRaWanNetworkSrvModule: Response should be a base64 encoded credential sent as plaintext
 
 ```
 
 In addition to this, the Azure Function must be able to properly authenticate to target sink via Managed Identity (when using Azure Blob Storage or KeyVault).
 
+For the very first version of the implementation, only Azure Storage Account should be implemented and only connection string authentication is going to be supported.
```

Comment:
Connection string for auth Function-> storage right? 
Would it be really harder to use MI straight away?

Reply:
Not harder, but requires more changes in the templates